### ISSUE TITLE
shh.getFilterMessages  sometimes did not get the post data

### ISSUE THREAD

**Issue by gongniuxiaodong** [ethereum/web3.js#1059]
sometimes logs is empty,  sometimes get right payload. it seem like  getFilterMessage can't get payload in time.
```js
var id='494d4f92f62008f826bd71baa7e24b0b1b1bc100832a7a7fdaa2979184127282';
var mesFilter=shh.newMessageFilter({symKeyID:id,topic:topic});console.log(mesFilter)
shh.post({symKeyID:id,ttl:7 ,topic:topic, payload:data,powTarget:2.01,powTime:2},function (e,r) {
    if(e){console.log(e)}
    else{console.log(r)}
});
function read() {
    var logs=shh.getFilterMessages(mesFilter)
    console.log(logs)
}
setInterval(read,2000)
```

**Comment by frozeman** [comment]
@gballet any idea? This is in web3.js 1.0 i suppose? @gongniuxiaodong ?

**Comment by gballet** [comment]
@frozeman This is fixed by #15165.
